cmd/gpu_plugin/gpu_plugin.go

```diff
@@ -723,6 +766,7 @@ func main() {
 	flag.IntVar(&opts.sharedDevNum, "shared-dev-num", 1, "number of containers sharing the same GPU device")
 	flag.IntVar(&opts.temperatureLimit, "temp-limit", 100, "temperature limit at which device is marked unhealthy")
 	flag.StringVar(&opts.preferredAllocationPolicy, "allocation-policy", "none", "modes of allocating GPU devices: balanced, packed and none")
+	flag.StringVar(&opts.allowlistIDs, "allowlist-ids", "", "comma-separated list of device IDs to allow (e.g. 0x49c5,0x49c6)")
 	flag.Parse()
```

Comment:
Consider adding also opposite option:
```suggestion
	flag.StringVar(&opts.ignoreIDs, "ignore-ids", "", "comma-separated list of device IDs to ignore (e.g. 0x49c5,0x49c6)")
	flag.Parse()
```
In some cases it's easier to maintain list what should be ignored, than what should be considered, especially when HW is updated / added.

It can use same parsing function, and the other code for ID checks could be also generalized to helper functions that work for both purposes, so needed extra code is mainly just for tests.

Note: If both options are given, there should be an error asking user to select either of them.